Source/WebKit/Configurations/BaseExtension.xcconfig

```diff
@@ -30,7 +30,7 @@ INSTALL_PATH = $(INSTALL_PATH_PREFIX)/System/Library/ExtensionKit/Extensions;
 INSTALL_PATH[sdk=*simulator] = $(WK_INSTALL_PATH_PREFIX)/System/Library/ExtensionKit/Extensions;
 INSTALL_PATH[sdk=iphone*17.4*] = $(WK_INSTALL_PATH_PREFIX)/System/Library/ExtensionKit/Extensions;
 SKIP_INSTALL = YES;
-SKIP_INSTALL[sdk=iphone*] = NO;
+SKIP_INSTALL[sdk=ios*] = NO;
```

Comment:
To be clear, this actually doesn't work AFAIK, my r+ is for the previous iteration.

Reply:
Sounds good! I have gone back to the previous iteration in the latest patch.

Thanks for approving!